src/data-card-advanced.tsx

```diff
@@ -0,0 +1,498 @@
+                        aria-label="Texto"
+                    >
+                        {description}
+                    </Text2>
+                )}
+            </Stack>
+        </div>
+    );
+};
+
+type CardFooterProps = {
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+};
+
+const CardFooter: React.FC<CardFooterProps> = ({
+    button,
+    footerImage,
+    footerText,
+    footerTextLinesMax,
+    buttonLink,
+}) => {
+    const {isMobile, isTabletOrBigger} = useScreenSize();
+
+    const hasButton = !!button;
+    const hasFooterImage = !!footerImage;
+    const hasFooterText = !!footerText;
+    const hasButtonLink = !!buttonLink;
+    const hasAllItens = hasButton && (hasFooterImage || hasFooterText) && hasButtonLink;
+
+    const footerCondition = hasAllItens && isMobile;
+
+    const flexDirection = footerCondition ? 'column' : 'row';
+    const marginRight = footerCondition ? '' : 'auto';
+    const alignItems = footerCondition ? 'start' : 'center';
+    const marginTop = hasAllItens ? '8px' : '16px';
+    const marginTopTabletOrBigger = isTabletOrBigger ? '8px' : '';
+    const marginTopButton = isMobile ? '16px' : '8px';
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <div>
+            <div style={{marginLeft: -24, marginRight: -24}}>
+                <Divider />
+            </div>
+
+            <div
+                className={styles.actions}
+                style={{flexDirection, alignItems, marginTop: marginTopTabletOrBigger}}
+            >
+                {hasButton && (
+                    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+                    <div
+                        aria-label="button"
+                        tabIndex={-1}
+                        onClick={(event) => event.stopPropagation()}
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        style={{
+                            marginTop: marginTopButton,
+                            marginRight: '16px',
+                            position: 'relative',
+                            zIndex: '2',
+                        }}
+                    >
+                        {button}
+                    </div>
+                )}
+                <div
+                    style={{
+                        display: 'flex',
+                        flexDirection: 'row',
+                        marginTop,
+                        flexWrap: 'wrap',
+                        marginRight,
+                    }}
+                >
+                    {hasFooterImage && (
+                        <div
+                            style={{marginRight: '16px', zIndex: '0'}}
```

Comment:
why zindex 0?

Reply:
The image component, inside the touchable doesn't become clickable, but with zindex: 0, it becomes clickable